retirement_api/utils/ss_calculator.py

```diff
@@ -417,20 +414,37 @@ def parse_response(results, html, language):
     return (results, base_benefit)
 
 
+def validate_date(params):
+    """Make sure delivered date is real"""
+    dobstring = "{0}-{1}-{2}".format(params['yob'],
+                                     params['dobmon'],
+                                     params['dobday'])
+    try:
+        dob = parser.parse(dobstring).date()
+        LOGGER.info("{} is valid".format(dob))
```

Comment:
Minor request: can you please remove these logging statements? Having birthdates as part of server logs is probably not desirable.